src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=""
+      data-qa="big"
+      method="GET"
+      class="margin"
```

Comment:
```suggestion
      class="form"
```
it would be better to not name classes by properties in this case, what if you need to add other properties to this class?)